2.3.0 failures on arm64 (test_doctests & test_geometry_area_perimeter__polygon__holes)

The Debian package build for pyproj 2.3.0 failed on arm64 due to test failures:
```
I: pybuild base:217: cd /<<PKGBUILDDIR>>/.pybuild/cpython3_3.7_pyproj/build; python3.7 -m pytest -v --ignore=test/test_datadir.py
============================= test session starts ==============================
platform linux -- Python 3.7.4, pytest-4.6.5, py-1.8.0, pluggy-0.12.0 -- /usr/bin/python3.7
cachedir: .pytest_cache
rootdir: /<<PKGBUILDDIR>>
plugins: cov-2.7.1
collecting ... collected 242 items

test/test_awips221.py::test_awips221 PASSED                              [  0%]
test/test_crs.py::test_from_proj4_json PASSED                            [  0%]
test/test_crs.py::test_from_proj4 PASSED                                 [  1%]
test/test_crs.py::test_from_proj4__invalid PASSED                        [  1%]
test/test_crs.py::test_from_epsg PASSED                                  [  2%]
test/test_crs.py::test_from_epsg_string PASSED                           [  2%]
test/test_crs.py::test_from_string PASSED                                [  2%]
test/test_crs.py::test_bare_parameters PASSED                            [  3%]
test/test_crs.py::test_is_geographic PASSED                              [  3%]
test/test_crs.py::test_is_projected PASSED                               [  4%]
test/test_crs.py::test_is_same_crs PASSED                                [  4%]
test/test_crs.py::test_to_proj4 PASSED                                   [  4%]
test/test_crs.py::test_empty_json PASSED                                 [  5%]
test/test_crs.py::test_has_wkt_property PASSED                           [  5%]
test/test_crs.py::test_to_wkt_pretty PASSED                              [  6%]
test/test_crs.py::test_repr PASSED                                       [  6%]
test/test_crs.py::test_repr__long PASSED                                 [  7%]
test/test_crs.py::test_repr_epsg PASSED                                  [  7%]
test/test_crs.py::test_repr__undefined PASSED                            [  7%]
test/test_crs.py::test_repr_compound PASSED                              [  8%]
test/test_crs.py::test_dunder_str PASSED                                 [  8%]
test/test_crs.py::test_epsg PASSED                                       [  9%]
test/test_crs.py::test_datum PASSED                                      [  9%]
test/test_crs.py::test_datum_horizontal PASSED                           [  9%]
test/test_crs.py::test_datum_unknown PASSED                              [ 10%]
test/test_crs.py::test_epsg__not_found PASSED                            [ 10%]
test/test_crs.py::test_epsg__no_code_available PASSED                    [ 11%]
test/test_crs.py::test_crs_OSR_equivalence PASSED                        [ 11%]
test/test_crs.py::test_crs_OSR_no_equivalence PASSED                     [ 11%]
test/test_crs.py::test_init_from_wkt PASSED                              [ 12%]
test/test_crs.py::test_init_from_wkt_invalid PASSED                      [ 12%]
test/test_crs.py::test_from_wkt PASSED                                   [ 13%]
test/test_crs.py::test_from_wkt_invalid PASSED                           [ 13%]
test/test_crs.py::test_from_user_input_epsg PASSED                       [ 14%]
test/test_crs.py::test_from_esri_wkt PASSED                              [ 14%]
test/test_crs.py::test_compound_crs PASSED                               [ 14%]
test/test_crs.py::test_ellipsoid PASSED                                  [ 15%]
test/test_crs.py::test_area_of_use PASSED                                [ 15%]
test/test_crs.py::test_from_user_input_custom_crs_class PASSED           [ 16%]
test/test_crs.py::test_non_crs_error PASSED                              [ 16%]
test/test_crs.py::test_sub_crs PASSED                                    [ 16%]
test/test_crs.py::test_sub_crs__none PASSED                              [ 17%]
test/test_crs.py::test_coordinate_system PASSED                          [ 17%]
test/test_crs.py::test_coordinate_system_geog PASSED                     [ 18%]
test/test_crs.py::test_coordinate_operation PASSED                       [ 18%]
test/test_crs.py::test_coordinate_operation_grids PASSED                 [ 19%]
test/test_crs.py::test_coordinate_operation_grids__alternative_grid_name PASSED [ 19%]
test/test_crs.py::test_coordinate_operation__missing PASSED              [ 19%]
test/test_crs.py::test_coordinate_operation__from_epsg PASSED            [ 20%]
test/test_crs.py::test_coordinate_operation__from_authority PASSED       [ 20%]
test/test_crs.py::test_coordinate_operation__from_epsg__empty PASSED     [ 21%]
test/test_crs.py::test_coordinate_operation__from_authority__empty PASSED [ 21%]
test/test_crs.py::test_datum__from_epsg PASSED                           [ 21%]
test/test_crs.py::test_datum__from_authority PASSED                      [ 22%]
test/test_crs.py::test_datum__from_epsg__invalid PASSED                  [ 22%]
test/test_crs.py::test_datum__from_authority__invalid PASSED             [ 23%]
test/test_crs.py::test_prime_meridian__from_epsg PASSED                  [ 23%]
test/test_crs.py::test_prime_meridian__from_authority PASSED             [ 23%]
test/test_crs.py::test_prime_meridian__from_epsg__invalid PASSED         [ 24%]
test/test_crs.py::test_prime_meridian__from_authority__invalid PASSED    [ 24%]
test/test_crs.py::test_ellipsoid__from_epsg PASSED                       [ 25%]
test/test_crs.py::test_ellipsoid__from_authority PASSED                  [ 25%]
test/test_crs.py::test_ellipsoid__from_epsg__invalid PASSED              [ 26%]
test/test_crs.py::test_ellipsoid__from_authority__invalid PASSED         [ 26%]
test/test_crs.py::test_bound_crs_is_geographic PASSED                    [ 26%]
test/test_crs.py::test_coordinate_operation_towgs84_three PASSED         [ 27%]
test/test_crs.py::test_coordinate_operation_towgs84_seven PASSED         [ 27%]
test/test_crs.py::test_coordinate_operation_towgs84_missing PASSED       [ 28%]
test/test_crs.py::test_to_wkt_enum PASSED                                [ 28%]
test/test_crs.py::test_to_wkt_enum__invalid PASSED                       [ 28%]
test/test_crs.py::test_to_proj4_enum PASSED                              [ 29%]
test/test_crs.py::test_datum__from_string PASSED                         [ 29%]
test/test_crs.py::test_datum__from_string__invalid PASSED                [ 30%]
test/test_crs.py::test_ellipsoid__from_string PASSED                     [ 30%]
test/test_crs.py::test_ellipsoid__from_string__invalid PASSED            [ 30%]
test/test_crs.py::test_prime_meridian__from_string PASSED                [ 31%]
test/test_crs.py::test_prime_meridian__from_string__invalid PASSED       [ 31%]
test/test_crs.py::test_coordinate_operation__from_string PASSED          [ 32%]
test/test_crs.py::test_coordinate_operation__from_string__invalid PASSED [ 32%]
test/test_crs.py::test_to_proj4_enum__invalid PASSED                     [ 33%]
test/test_crs.py::test_geodetic_crs PASSED                               [ 33%]
test/test_crs.py::test_itrf_init PASSED                                  [ 33%]
test/test_crs.py::test_compound_crs_init PASSED                          [ 34%]
test/test_crs.py::test_compound_crs_urn_init PASSED                      [ 34%]
test/test_crs.py::test_from_authority__ignf PASSED                       [ 35%]
test/test_crs.py::test_ignf_authority_repr PASSED                        [ 35%]
test/test_crs.py::test_crs_hash PASSED                                   [ 35%]
test/test_crs.py::test_crs_hash_unequal PASSED                           [ 36%]
test/test_crs.py::test_crs_init_user_input PASSED                        [ 36%]
test/test_crs.py::test_to_string__no_auth PASSED                         [ 37%]
test/test_crs.py::test_to_string__auth PASSED                            [ 37%]
test/test_crs.py::test_srs__no_plus PASSED                               [ 38%]
test/test_crs.py::test_equals_different_type PASSED                      [ 38%]
test/test_crs.py::test_is_exact_same_different_type PASSED               [ 38%]
test/test_crs.py::test_compare_crs_non_crs PASSED                        [ 39%]
test/test_crs.py::test_is_geocentric__bound PASSED                       [ 39%]
test/test_crs.py::test_is_geocentric PASSED                              [ 40%]
test/test_crs.py::test_is_vertical PASSED                                [ 40%]
test/test_crs.py::test_is_engineering PASSED                             [ 40%]
test/test_crs.py::test_source_crs__bound PASSED                          [ 41%]
test/test_crs.py::test_source_crs__missing PASSED                        [ 41%]
test/test_crs.py::test_target_crs__bound PASSED                          [ 42%]
test/test_crs.py::test_target_crs__missing PASSED                        [ 42%]
test/test_crs.py::test_whitepace_between_equals PASSED                   [ 42%]
test/test_crs.py::test_to_dict_no_proj4 PASSED                           [ 43%]
test/test_crs_cf.py::test_to_cf_transverse_mercator PASSED               [ 43%]
test/test_crs_cf.py::test_from_cf_transverse_mercator PASSED             [ 44%]
test/test_crs_cf.py::test_cf_from_latlon PASSED                          [ 44%]
test/test_crs_cf.py::test_cf_from_latlon__named PASSED                   [ 45%]
test/test_crs_cf.py::test_cf_from_utm PASSED                             [ 45%]
test/test_crs_cf.py::test_cf_from_utm__nad83 PASSED                      [ 45%]
test/test_crs_cf.py::test_cf_rotated_latlon PASSED                       [ 46%]
test/test_crs_cf.py::test_cf_rotated_latlon__grid PASSED                 [ 46%]
test/test_crs_cf.py::test_cf_lambert_conformal_conic PASSED              [ 47%]
test/test_crs_cf.py::test_cf_lambert_conformal_conic_1sp PASSED          [ 47%]
test/test_crs_cf.py::test_cf_lambert_conformal_conic_2sp PASSED          [ 47%]
test/test_crs_cf.py::test_oblique_mercator PASSED                        [ 48%]
test/test_crs_cf.py::test_cf_from_invalid PASSED                         [ 48%]
test/test_crs_cf.py::test_crs_sweep PASSED                               [ 49%]
test/test_crs_cf.py::test_crs_fixed_angle_axis PASSED                    [ 49%]
test/test_crs_cf.py::test_ob_tran_not_rotated_latlon PASSED              [ 50%]
test/test_datum.py::test_datum PASSED                                    [ 50%]
test/test_datum_shift.py::test_shift_wgs84_to_utm33 PASSED               [ 50%]
test/test_datum_shift.py::test_shift_utm33_to_wgs84 PASSED               [ 51%]
test/test_datum_shift.py::test_shift_wgs84_to_gaussb_no_ellisoidal_height PASSED [ 51%]
test/test_datum_shift.py::test_shift_gaussb_to_wgs84_no_ellisoidal_height PASSED [ 52%]
test/test_doctest_wrapper.py::test_doctests FAILED                       [ 52%]
test/test_exception_logging.py::test_proj_exception PASSED               [ 52%]
test/test_exception_logging.py::test_crs_exception PASSED                [ 53%]
test/test_geod.py::test_geod_inverse_transform PASSED                    [ 53%]
test/test_geod.py::test_geod_cities PASSED                               [ 54%]
test/test_geod.py::test_line_length__single_point PASSED                 [ 54%]
test/test_geod.py::test_line_length__radians PASSED                      [ 54%]
test/test_geod.py::test_line_lengths__single_point PASSED                [ 55%]
test/test_geod.py::test_line_lengths__radians PASSED                     [ 55%]
test/test_geod.py::test_polygon_area_perimeter__single_point PASSED      [ 56%]
test/test_geod.py::test_geometry_length__point PASSED                    [ 56%]
test/test_geod.py::test_geometry_length__linestring PASSED               [ 57%]
test/test_geod.py::test_geometry_length__linestring__radians PASSED      [ 57%]
test/test_geod.py::test_geometry_length__linearring PASSED               [ 57%]
test/test_geod.py::test_geometry_length__polygon PASSED                  [ 58%]
test/test_geod.py::test_geometry_length__polygon__radians PASSED         [ 58%]
test/test_geod.py::test_geometry_length__multipolygon PASSED             [ 59%]
test/test_geod.py::test_geometry_length__multipolygon__radians PASSED    [ 59%]
test/test_geod.py::test_geometry_length__multilinestring PASSED          [ 59%]
test/test_geod.py::test_geometry_length__multipoint PASSED               [ 60%]
test/test_geod.py::test_geometry_area_perimeter__point PASSED            [ 60%]
test/test_geod.py::test_geometry_area_perimeter__linestring PASSED       [ 61%]
test/test_geod.py::test_geometry_area_perimeter__linestring__radians PASSED [ 61%]
test/test_geod.py::test_geometry_area_perimeter__linearring PASSED       [ 61%]
test/test_geod.py::test_geometry_area_perimeter__polygon PASSED          [ 62%]
test/test_geod.py::test_geometry_area_perimeter__polygon__radians PASSED [ 62%]
test/test_geod.py::test_geometry_area_perimeter__polygon__holes FAILED   [ 63%]
test/test_geod.py::test_geometry_area_perimeter__multipolygon PASSED     [ 63%]
test/test_geod.py::test_geometry_area_perimeter__multipolygon__radians PASSED [ 64%]
test/test_geod.py::test_geometry_area_perimeter__multilinestring PASSED  [ 64%]
test/test_geod.py::test_geometry_area_perimeter__multipoint PASSED       [ 64%]
test/test_list.py::test_units_map PASSED                                 [ 65%]
test/test_list.py::test_angular_units_map PASSED                         [ 65%]
test/test_list.py::test_get_ellps_map PASSED                             [ 66%]
test/test_list.py::test_get_prime_meridians_map PASSED                   [ 66%]
test/test_list.py::test_get_proj_operations_map PASSED                   [ 66%]
test/test_pickle.py::test_pickle PASSED                                  [ 67%]
test/test_proj.py::BasicTest::testInitWithBackupString4 PASSED           [ 67%]
test/test_proj.py::BasicTest::testProjAwips221 PASSED                    [ 68%]
test/test_proj.py::BasicTest::test_from_dict_with_bool PASSED            [ 68%]
test/test_proj.py::InverseHammerTest::test_forward PASSED                [ 69%]
test/test_proj.py::InverseHammerTest::test_inverse PASSED                [ 69%]
test/test_proj.py::TypeError_Transform_Issue8_Test::test_tranform_none_1st_parmeter PASSED [ 69%]
test/test_proj.py::TypeError_Transform_Issue8_Test::test_tranform_none_2nd_parmeter PASSED [ 70%]
test/test_proj.py::Geod_NoDefs_Issue22_Test::test_geod_nodefs PASSED     [ 70%]
test/test_proj.py::ProjLatLongTypeErrorTest::test_latlong_typeerror PASSED [ 71%]
test/test_proj.py::ForwardInverseTest::test_fwd_inv PASSED               [ 71%]
test/test_proj.py::GeodSharedMemoryBugTestIssue64::test_distances PASSED [ 71%]
test/test_proj.py::GeodSharedMemoryBugTestIssue64::test_not_shared_memory PASSED [ 72%]
test/test_proj.py::ReprTests::test_ellps_name_round_trip PASSED          [ 72%]
test/test_proj.py::ReprTests::test_repr PASSED                           [ 73%]
test/test_proj.py::ReprTests::test_sphere PASSED                         [ 73%]
test/test_proj.py::TestRadians::test_fwd_radians PASSED                  [ 73%]
test/test_proj.py::TestRadians::test_inv_radians PASSED                  [ 74%]
test/test_proj.py::TestRadians::test_npts_radians PASSED                 [ 74%]
test/test_proj.py::Geod_NaN_Issue112_Test::test_geod_nans PASSED         [ 75%]
test/test_proj.py::test_proj_equals PASSED                               [ 75%]
test/test_proj.py::test_initialize_proj_crs_no_proj4 PASSED              [ 76%]
test/test_proj.py::test_initialize_proj_crs_no_plus PASSED               [ 76%]
test/test_proj.py::test_equals_different_type PASSED                     [ 76%]
test/test_proj.py::test_is_exact_same_different_type PASSED              [ 77%]
test/test_show_versions.py::test_get_sys_info PASSED                     [ 77%]
test/test_show_versions.py::test_get_deps_info PASSED                    [ 78%]
test/test_show_versions.py::test_show_versions_with_proj PASSED          [ 78%]
test/test_transform.py::test_transform PASSED                            [ 78%]
test/test_transformer.py::test_tranform_wgs84_to_custom PASSED           [ 79%]
test/test_transformer.py::test_transform_wgs84_to_alaska PASSED          [ 79%]
test/test_transformer.py::test_illegal_transformation PASSED             [ 80%]
test/test_transformer.py::test_lambert_conformal_transform PASSED        [ 80%]
test/test_transformer.py::test_equivalent_crs PASSED                     [ 80%]
test/test_transformer.py::test_equivalent_crs__disabled PASSED           [ 81%]
test/test_transformer.py::test_equivalent_crs__different PASSED          [ 81%]
test/test_transformer.py::test_equivalent_proj PASSED                    [ 82%]
test/test_transformer.py::test_equivalent_proj__disabled PASSED          [ 82%]
test/test_transformer.py::test_equivalent_proj__different PASSED         [ 83%]
test/test_transformer.py::test_equivalent_pipeline PASSED                [ 83%]
test/test_transformer.py::test_4d_transform PASSED                       [ 83%]
test/test_transformer.py::test_2d_with_time_transform PASSED             [ 84%]
test/test_transformer.py::test_4d_transform_crs_obs1 PASSED              [ 84%]
test/test_transformer.py::test_4d_transform_orginal_crs_obs1 PASSED      [ 85%]
test/test_transformer.py::test_4d_transform_crs_obs2 PASSED              [ 85%]
test/test_transformer.py::test_2d_with_time_transform_crs_obs2 PASSED    [ 85%]
test/test_transformer.py::test_2d_with_time_transform_original_crs_obs2 PASSED [ 86%]
test/test_transformer.py::test_4d_itransform PASSED                      [ 86%]
test/test_transformer.py::test_3d_time_itransform PASSED                 [ 87%]
test/test_transformer.py::test_4d_itransform_orginal_crs_obs1 PASSED     [ 87%]
test/test_transformer.py::test_2d_with_time_itransform_original_crs_obs2 PASSED [ 88%]
test/test_transformer.py::test_itransform_time_3rd_invalid PASSED        [ 88%]
test/test_transformer.py::test_transform_no_error PASSED                 [ 88%]
test/test_transformer.py::test_itransform_no_error PASSED                [ 89%]
test/test_transformer.py::test_transform_no_exception PASSED             [ 89%]
test/test_transformer.py::test_transform__out_of_bounds PASSED           [ 90%]
test/test_transformer.py::test_transform_radians PASSED                  [ 90%]
test/test_transformer.py::test_itransform_radians PASSED                 [ 90%]
test/test_transformer.py::test_4d_transform__inverse PASSED              [ 91%]
test/test_transformer.py::test_transform_direction PASSED                [ 91%]
test/test_transformer.py::test_always_xy__transformer PASSED             [ 92%]
test/test_transformer.py::test_always_xy__transform PASSED               [ 92%]
test/test_transformer.py::test_always_xy__itransform PASSED              [ 92%]
test/test_transformer.py::test_transform_direction__string PASSED        [ 93%]
test/test_transformer.py::test_transform_direction__string_lowercase PASSED [ 93%]
test/test_transformer.py::test_transform_direction__invalid PASSED       [ 94%]
test/test_transformer.py::test_from_pipeline__non_transform_input PASSED [ 94%]
test/test_transformer.py::test_non_supported_initialization PASSED       [ 95%]
test/test_transformer.py::test_pj_info_properties PASSED                 [ 95%]
test/test_transformer.py::test_to_wkt PASSED                             [ 95%]
test/test_transformer.py::test_str PASSED                                [ 96%]
test/test_transformer.py::test_repr PASSED                               [ 96%]
test/test_transformer.py::test_transformer_group PASSED                  [ 97%]
test/test_transformer.py::test_transformer_group__unavailable PASSED     [ 97%]
test/test_transformer.py::test_transform_group__missing_best PASSED      [ 97%]
test/test_transformer.py::test_transform_group__area_of_interest PASSED  [ 98%]
test/test_transformer.py::test_transformer__area_of_interest PASSED      [ 98%]
test/test_transformer.py::test_transformer_proj__area_of_interest PASSED [ 99%]
test/test_transformer.py::test_transformer__area_of_interest__invalid PASSED [ 99%]
test/test_transformer.py::test_transformer_group__area_of_interest__invalid PASSED [100%]

=================================== FAILURES ===================================
________________________________ test_doctests _________________________________

    def test_doctests():
        failure_count = pyproj.test()
    
        # shapely wheels not on windows, so allow failures there
        expected_failure_count = 0
        try:
            import shapely  # noqa
        except ImportError:
            if os.name == "nt":
                expected_failure_count = 6
    
        # if the below line fails, doctests have failed
>       assert (
            failure_count == expected_failure_count
        ), "{0} of the doctests in " "lib/pyproj/__init__.py failed".format(failure_count)
E       AssertionError: 1 of the doctests in lib/pyproj/__init__.py failed
E       assert 1 == 0
E         -1
E         +0

test/test_doctest_wrapper.py:22: AssertionError
----------------------------- Captured stdout call -----------------------------
Trying:
    from pyproj import Proj
Expecting nothing
ok
Trying:
    p = Proj(proj='utm',zone=10,ellps='WGS84', preserve_units=False)
Expecting nothing
ok
Trying:
    x,y = p(-120.108, 34.36116666)
Expecting nothing
ok
Trying:
    'x=%9.3f y=%11.3f' % (x,y)
Expecting:
    'x=765975.641 y=3805993.134'
ok
Trying:
    'lon=%8.3f lat=%5.3f' % p(x,y,inverse=True)
Expecting:
    'lon=-120.108 lat=34.361'
ok
Trying:
    lons = (-119.72,-118.40,-122.38)
Expecting nothing
ok
Trying:
    lats = (36.77, 33.93, 37.62 )
Expecting nothing
ok
Trying:
    x,y = p(lons, lats)
Expecting nothing
ok
Trying:
    'x: %9.3f %9.3f %9.3f' % x
Expecting:
    'x: 792763.863 925321.537 554714.301'
ok
Trying:
    'y: %9.3f %9.3f %9.3f' % y
Expecting:
    'y: 4074377.617 3763936.941 4163835.303'
ok
Trying:
    lons, lats = p(x, y, inverse=True) # inverse transform
Expecting nothing
ok
Trying:
    'lons: %8.3f %8.3f %8.3f' % lons
Expecting:
    'lons: -119.720 -118.400 -122.380'
ok
Trying:
    'lats: %8.3f %8.3f %8.3f' % lats
Expecting:
    'lats:   36.770   33.930   37.620'
ok
Trying:
    p2 = Proj('+proj=utm +zone=10 +ellps=WGS84', preserve_units=False)
Expecting nothing
ok
Trying:
    x,y = p2(-120.108, 34.36116666)
Expecting nothing
ok
Trying:
    'x=%9.3f y=%11.3f' % (x,y)
Expecting:
    'x=765975.641 y=3805993.134'
ok
Trying:
    p = Proj("epsg:32667", preserve_units=False)
Expecting nothing
ok
Trying:
    'x=%12.3f y=%12.3f (meters)' % p(-114.057222, 51.045)
Expecting:
    'x=-1783506.250 y= 6193827.033 (meters)'
ok
Trying:
    p = Proj("epsg:32667")
Expecting nothing
ok
Trying:
    'x=%12.3f y=%12.3f (feet)' % p(-114.057222, 51.045)
Expecting:
    'x=-5851386.754 y=20320914.191 (feet)'
ok
Trying:
    p1 = Proj("epsg:4214")
Expecting nothing
ok
Trying:
    x1, y1 = p1(116.366, 39.867)
Expecting nothing
ok
Trying:
    '{:.3f} {:.3f}'.format(x1, y1)
Expecting:
    '116.366 39.867'
ok
Trying:
    x2, y2 = p1(x1, y1, inverse=True)
Expecting nothing
ok
Trying:
    '{:.3f} {:.3f}'.format(x2, y2)
Expecting:
    '116.366 39.867'
ok
Trying:
    Proj("epsg:4326").definition_string()
Expecting:
    'proj=longlat datum=WGS84 no_defs ellps=WGS84 towgs84=0,0,0'
ok
5 items had no tests:
    pyproj.proj
    pyproj.proj.Proj
    pyproj.proj.Proj.__call__
    pyproj.proj.Proj.to_latlong
    pyproj.proj.Proj.to_latlong_def
2 items passed all tests:
  25 tests in pyproj.proj.Proj.__init__
   1 tests in pyproj.proj.Proj.definition_string
26 tests in 7 items.
26 passed and 0 failed.
Test passed.
Trying:
    from pyproj import CRS
Expecting nothing
ok
Trying:
    crs_utm = CRS.from_user_input(26915)
Expecting nothing
ok
Trying:
    crs_utm
Expecting:
    <Projected CRS: EPSG:26915>
    Name: NAD83 / UTM zone 15N
    Axis Info [cartesian]:
    - E[east]: Easting (metre)
    - N[north]: Northing (metre)
    Area of Use:
    - name: North America - 96°W to 90°W and NAD83 by country
    - bounds: (-96.0, 25.61, -90.0, 84.0)
    Coordinate Operation:
    - name: UTM zone 15N
    - method: Transverse Mercator
    Datum: North American Datum 1983
    - Ellipsoid: GRS 1980
    - Prime Meridian: Greenwich
    <BLANKLINE>
ok
Trying:
    crs_utm.area_of_use.bounds
Expecting:
    (-96.0, 25.61, -90.0, 84.0)
ok
Trying:
    crs_utm.ellipsoid
Expecting:
    ELLIPSOID["GRS 1980",6378137,298.257222101,
        LENGTHUNIT["metre",1],
        ID["EPSG",7019]]
ok
Trying:
    crs_utm.ellipsoid.inverse_flattening
Expecting:
    298.257222101
ok
Trying:
    crs_utm.ellipsoid.semi_major_metre
Expecting:
    6378137.0
ok
Trying:
    crs_utm.ellipsoid.semi_minor_metre
Expecting:
    6356752.314140356
ok
Trying:
    crs_utm.prime_meridian
Expecting:
    PRIMEM["Greenwich",0,
        ANGLEUNIT["degree",0.0174532925199433],
        ID["EPSG",8901]]
ok
Trying:
    crs_utm.prime_meridian.unit_name
Expecting:
    'degree'
ok
Trying:
    crs_utm.prime_meridian.unit_conversion_factor
Expecting:
    0.017453292519943295
ok
Trying:
    crs_utm.prime_meridian.longitude
Expecting:
    0.0
ok
Trying:
    crs_utm.datum
Expecting:
    DATUM["North American Datum 1983",
        ELLIPSOID["GRS 1980",6378137,298.257222101,
            LENGTHUNIT["metre",1]],
        ID["EPSG",6269]]
ok
Trying:
    crs_utm.coordinate_system
Expecting:
    CS[Cartesian,2],
        AXIS["(E)",east,
            ORDER[1],
            LENGTHUNIT["metre",1,
                ID["EPSG",9001]]],
        AXIS["(N)",north,
            ORDER[2],
            LENGTHUNIT["metre",1,
                ID["EPSG",9001]]]
ok
Trying:
    crs_utm.coordinate_operation
Expecting:
    CONVERSION["UTM zone 15N",
        METHOD["Transverse Mercator",
            ID["EPSG",9807]],
        PARAMETER["Latitude of natural origin",0,
            ANGLEUNIT["degree",0.0174532925199433],
            ID["EPSG",8801]],
        PARAMETER["Longitude of natural origin",-93,
            ANGLEUNIT["degree",0.0174532925199433],
            ID["EPSG",8802]],
        PARAMETER["Scale factor at natural origin",0.9996,
            SCALEUNIT["unity",1],
            ID["EPSG",8805]],
        PARAMETER["False easting",500000,
            LENGTHUNIT["metre",1],
            ID["EPSG",8806]],
        PARAMETER["False northing",0,
            LENGTHUNIT["metre",1],
            ID["EPSG",8807]],
        ID["EPSG",16015]]
ok
Trying:
    crs = CRS(proj='utm', zone=10, ellps='WGS84')
Expecting nothing
ok
Trying:
    crs.to_proj4()
Expecting:
    '+proj=utm +zone=10 +ellps=WGS84 +units=m +no_defs +type=crs'
ok
Trying:
    print(crs.to_wkt(pretty=True))
Expecting:
    PROJCRS["unknown",
        BASEGEOGCRS["unknown",
            DATUM["Unknown based on WGS84 ellipsoid",
                ELLIPSOID["WGS 84",6378137,298.257223563,
                    LENGTHUNIT["metre",1],
                    ID["EPSG",7030]]],
            PRIMEM["Greenwich",0,
                ANGLEUNIT["degree",0.0174532925199433],
                ID["EPSG",8901]]],
        CONVERSION["UTM zone 10N",
            METHOD["Transverse Mercator",
                ID["EPSG",9807]],
            PARAMETER["Latitude of natural origin",0,
                ANGLEUNIT["degree",0.0174532925199433],
                ID["EPSG",8801]],
            PARAMETER["Longitude of natural origin",-123,
                ANGLEUNIT["degree",0.0174532925199433],
                ID["EPSG",8802]],
            PARAMETER["Scale factor at natural origin",0.9996,
                SCALEUNIT["unity",1],
                ID["EPSG",8805]],
            PARAMETER["False easting",500000,
                LENGTHUNIT["metre",1],
                ID["EPSG",8806]],
            PARAMETER["False northing",0,
                LENGTHUNIT["metre",1],
                ID["EPSG",8807]],
            ID["EPSG",16010]],
        CS[Cartesian,2],
            AXIS["(E)",east,
                ORDER[1],
                LENGTHUNIT["metre",1,
                    ID["EPSG",9001]]],
            AXIS["(N)",north,
                ORDER[2],
                LENGTHUNIT["metre",1,
                    ID["EPSG",9001]]]]
ok
Trying:
    geod = crs.get_geod()
Expecting nothing
ok
Trying:
    "+a={:.0f} +f={:.8f}".format(geod.a, geod.f)
Expecting:
    '+a=6378137 +f=0.00335281'
ok
Trying:
    crs.is_projected
Expecting:
    True
ok
Trying:
    crs.is_geographic
Expecting:
    False
ok
23 items had no tests:
    pyproj.crs
    pyproj.crs.CRS
    pyproj.crs.CRS.__eq__
    pyproj.crs.CRS.__hash__
    pyproj.crs.CRS.__reduce__
    pyproj.crs.CRS.__repr__
    pyproj.crs.CRS.__str__
    pyproj.crs.CRS.from_authority
    pyproj.crs.CRS.from_cf
    pyproj.crs.CRS.from_dict
    pyproj.crs.CRS.from_epsg
    pyproj.crs.CRS.from_proj4
    pyproj.crs.CRS.from_string
    pyproj.crs.CRS.from_user_input
    pyproj.crs.CRS.from_wkt
    pyproj.crs.CRS.get_geod
    pyproj.crs.CRS.to_cf
    pyproj.crs.CRS.to_dict
    pyproj.crs.CRS.to_string
    pyproj.crs._prepare_from_authority
    pyproj.crs._prepare_from_dict
    pyproj.crs._prepare_from_epsg
    pyproj.crs._prepare_from_string
1 items passed all tests:
  22 tests in pyproj.crs.CRS.__init__
22 tests in 24 items.
22 passed and 0 failed.
Test passed.
Trying:
    from pyproj import Geod
Expecting nothing
ok
Trying:
    gclrk1 = Geod(ellps='clrk66')
Expecting nothing
ok
Trying:
    gclrk2 = Geod(a=6378206.4, b=6356583.8)
Expecting nothing
ok
Trying:
    gclrk1 == gclrk2
Expecting:
    True
ok
Trying:
    gwgs66 = Geod('+ellps=WGS66')
Expecting nothing
ok
Trying:
    gnwl9d = Geod('+ellps=NWL9D')
Expecting nothing
ok
Trying:
    gnwl9d == gwgs66
Expecting:
    True
ok
Trying:
    gclrk1 != gnwl9d  # Clarke 1866 is unlike NWL9D
Expecting:
    True
ok
Trying:
    from pyproj import Geod
Expecting nothing
ok
Trying:
    g = Geod(ellps='clrk66') # Use Clarke 1866 ellipsoid.
Expecting nothing
ok
Trying:
    boston_lat = 42.+(15./60.); boston_lon = -71.-(7./60.)
Expecting nothing
ok
Trying:
    portland_lat = 45.+(31./60.); portland_lon = -123.-(41./60.)
Expecting nothing
ok
Trying:
    newyork_lat = 40.+(47./60.); newyork_lon = -73.-(58./60.)
Expecting nothing
ok
Trying:
    london_lat = 51.+(32./60.); london_lon = -(5./60.)
Expecting nothing
ok
Trying:
    az12,az21,dist = g.inv(boston_lon,boston_lat,portland_lon,portland_lat)
Expecting nothing
ok
Trying:
    "%7.3f %6.3f %12.3f" % (az12,az21,dist)
Expecting:
    '-66.531 75.654  4164192.708'
ok
Trying:
    endlon, endlat, backaz = g.fwd(boston_lon, boston_lat, az12, dist)
Expecting nothing
ok
Trying:
    "%6.3f  %6.3f %13.3f" % (endlat,endlon,backaz)
Expecting:
    '45.517  -123.683        75.654'
ok
Trying:
    lons1 = 3*[newyork_lon]; lats1 = 3*[newyork_lat]
Expecting nothing
ok
Trying:
    lons2 = [boston_lon, portland_lon, london_lon]
Expecting nothing
ok
Trying:
    lats2 = [boston_lat, portland_lat, london_lat]
Expecting nothing
ok
Trying:
    az12,az21,dist = g.inv(lons1,lats1,lons2,lats2)
Expecting nothing
ok
Trying:
    for faz, baz, d in list(zip(az12,az21,dist)):
        "%7.3f %7.3f %9.3f" % (faz, baz, d)
Expecting:
    ' 54.663 -123.448 288303.720'
    '-65.463  79.342 4013037.318'
    ' 51.254 -71.576 5579916.651'
ok
Trying:
    g2 = Geod('+ellps=clrk66') # use proj4 style initialization string
Expecting nothing
ok
Trying:
    az12,az21,dist = g2.inv(boston_lon,boston_lat,portland_lon,portland_lat)
Expecting nothing
ok
Trying:
    "%7.3f %6.3f %12.3f" % (az12,az21,dist)
Expecting:
    '-66.531 75.654  4164192.708'
ok
Trying:
    from pyproj import Geod
Expecting nothing
ok
Trying:
    from shapely.geometry import LineString, Point, Polygon
Expecting nothing
ok
Trying:
    geod = Geod(ellps="WGS84")
Expecting nothing
ok
Trying:
    poly_area, poly_perimeter = geod.geometry_area_perimeter(
        Polygon(
            LineString([
                Point(1, 1), Point(1, 10), Point(10, 10), Point(10, 1)
            ]),
            holes=[LineString([Point(1, 2), Point(3, 4), Point(5, 2)])],
        )
    )
Expecting nothing
ok
Trying:
    "{:.3f} {:.3f}".format(poly_area, poly_perimeter)
Expecting:
    '-944373881400.339 3979008.036'
ok
Trying:
    from pyproj import Geod
Expecting nothing
ok
Trying:
    from shapely.geometry import Point, LineString
Expecting nothing
ok
Trying:
    line_string = LineString([Point(1, 2), Point(3, 4)])
Expecting nothing
ok
Trying:
    geod = Geod(ellps="WGS84")
Expecting nothing
ok
Trying:
    "{:.3f}".format(geod.geometry_length(line_string))
Expecting:
    '313588.397'
ok
Trying:
    from pyproj import Geod
Expecting nothing
ok
Trying:
    geod = Geod('+a=6378137 +f=0.0033528106647475126')
Expecting nothing
ok
Trying:
    lats = [-72.9, -71.9, -74.9, -74.3, -77.5, -77.4, -71.7, -65.9, -65.7,
            -66.6, -66.9, -69.8, -70.0, -71.0, -77.3, -77.9, -74.7]
Expecting nothing
ok
Trying:
    lons = [-74, -102, -102, -131, -163, 163, 172, 140, 113,
            88, 59, 25, -4, -14, -33, -46, -61]
Expecting nothing
ok
Trying:
    total_length = geod.line_length(lons, lats)
Expecting nothing
ok
Trying:
    "{:.3f}".format(total_length)
Expecting:
    '14259605.611'
ok
Trying:
    from pyproj import Geod
Expecting nothing
ok
Trying:
    geod = Geod(ellps="WGS84")
Expecting nothing
ok
Trying:
    lats = [-72.9, -71.9, -74.9]
Expecting nothing
ok
Trying:
    lons = [-74, -102, -102]
Expecting nothing
ok
Trying:
    for line_length in geod.line_lengths(lons, lats):
        "{:.3f}".format(line_length)
Expecting:
    '943065.744'
    '334805.010'
ok
Trying:
    from pyproj import Geod
Expecting nothing
ok
Trying:
    g = Geod(ellps='clrk66') # Use Clarke 1866 ellipsoid.
Expecting nothing
ok
Trying:
    boston_lat = 42.+(15./60.); boston_lon = -71.-(7./60.)
Expecting nothing
ok
Trying:
    portland_lat = 45.+(31./60.); portland_lon = -123.-(41./60.)
Expecting nothing
ok
Trying:
    lonlats = g.npts(boston_lon,boston_lat,portland_lon,portland_lat,10)
Expecting nothing
ok
Trying:
    for lon,lat in lonlats: '%6.3f  %7.3f' % (lat, lon)
Expecting:
    '43.528  -75.414'
    '44.637  -79.883'
    '45.565  -84.512'
    '46.299  -89.279'
    '46.830  -94.156'
    '47.149  -99.112'
    '47.251  -104.106'
    '47.136  -109.100'
    '46.805  -114.051'
    '46.262  -118.924'
ok
Trying:
    import math
Expecting nothing
ok
Trying:
    dg2rad = math.radians(1.)
Expecting nothing
ok
Trying:
    rad2dg = math.degrees(1.)
Expecting nothing
ok
Trying:
    lonlats = g.npts(
       dg2rad*boston_lon,
       dg2rad*boston_lat,
       dg2rad*portland_lon,
       dg2rad*portland_lat,
       10,
       radians=True
    )
Expecting nothing
ok
Trying:
    for lon,lat in lonlats: '%6.3f  %7.3f' % (rad2dg*lat, rad2dg*lon)
Expecting:
    '43.528  -75.414'
    '44.637  -79.883'
    '45.565  -84.512'
    '46.299  -89.279'
    '46.830  -94.156'
    '47.149  -99.112'
    '47.251  -104.106'
    '47.136  -109.100'
    '46.805  -114.051'
    '46.262  -118.924'
ok
Trying:
    from pyproj import Geod
Expecting nothing
ok
Trying:
    geod = Geod('+a=6378137 +f=0.0033528106647475126')
Expecting nothing
ok
Trying:
    lats = [-72.9, -71.9, -74.9, -74.3, -77.5, -77.4, -71.7, -65.9, -65.7,
            -66.6, -66.9, -69.8, -70.0, -71.0, -77.3, -77.9, -74.7]
Expecting nothing
ok
Trying:
    lons = [-74, -102, -102, -131, -163, 163, 172, 140, 113,
            88, 59, 25, -4, -14, -33, -46, -61]
Expecting nothing
ok
Trying:
    poly_area, poly_perimeter = geod.polygon_area_perimeter(lons, lats)
Expecting nothing
ok
Trying:
    "{:.3f} {:.3f}".format(poly_area, poly_perimeter)
Expecting:
    '13376856682207.406 14710425.407'
**********************************************************************
File "/<<PKGBUILDDIR>>/.pybuild/cpython3_3.7_pyproj/build/pyproj/geod.py", line 399, in pyproj.geod.Geod.polygon_area_perimeter
Failed example:
    "{:.3f} {:.3f}".format(poly_area, poly_perimeter)
Expected:
    '13376856682207.406 14710425.407'
Got:
    '13376856682207.375 14710425.407'
5 items had no tests:
    pyproj.geod
    pyproj.geod.Geod
    pyproj.geod.Geod.__repr__
    pyproj.geod.Geod.fwd
    pyproj.geod.Geod.inv
7 items passed all tests:
   8 tests in pyproj.geod.Geod.__eq__
  18 tests in pyproj.geod.Geod.__init__
   5 tests in pyproj.geod.Geod.geometry_area_perimeter
   5 tests in pyproj.geod.Geod.geometry_length
   6 tests in pyproj.geod.Geod.line_length
   5 tests in pyproj.geod.Geod.line_lengths
  11 tests in pyproj.geod.Geod.npts
**********************************************************************
1 items had failures:
   1 of   6 in pyproj.geod.Geod.polygon_area_perimeter
64 tests in 13 items.
63 passed and 1 failed.
***Test Failed*** 1 failures.
Trying:
    from pyproj import Transformer
Expecting nothing
ok
Trying:
    transformer = Transformer.from_crs(4326, 2100)
Expecting nothing
ok
Trying:
    points = [(22.95, 40.63), (22.81, 40.53), (23.51, 40.86)]
Expecting nothing
ok
Trying:
    for pt in transformer.itransform(points): '{:.3f} {:.3f}'.format(*pt)
Expecting:
    '2221638.801 2637034.372'
    '2212924.125 2619851.898'
    '2238294.779 2703763.736'
ok
Trying:
    pipeline_str = (
        "+proj=pipeline +step +proj=longlat +ellps=WGS84 "
        "+step +proj=unitconvert +xy_in=rad +xy_out=deg"
    )
Expecting nothing
ok
Trying:
    pipe_trans = Transformer.from_pipeline(pipeline_str)
Expecting nothing
ok
Trying:
    for pt in pipe_trans.itransform([(2.1, 0.001)]):
        '{:.3f} {:.3f}'.format(*pt)
Expecting:
    '120.321 0.057'
ok
Trying:
    transproj = Transformer.from_crs(
        {"proj":'geocent', "ellps":'WGS84', "datum":'WGS84'},
        "EPSG:4326",
        always_xy=True,
    )
Expecting nothing
ok
Trying:
    for pt in transproj.itransform(
        [(-2704026.010, -4253051.810, 3895878.820)],
        radians=True,
    ):
        '{:.3f} {:.3f} {:.3f}'.format(*pt)
Expecting:
    '-2.137 0.661 -20.531'
ok
Trying:
    transprojr = Transformer.from_crs(
        "EPSG:4326",
        {"proj":'geocent', "ellps":'WGS84', "datum":'WGS84'},
        always_xy=True,
    )
Expecting nothing
ok
Trying:
    for pt in transprojr.itransform(
        [(-2.137, 0.661, -20.531)],
        radians=True
    ):
        '{:.3f} {:.3f} {:.3f}'.format(*pt)
Expecting:
    '-2704214.394 -4254414.478 3894270.731'
ok
Trying:
    transproj_eq = Transformer.from_proj(
        'EPSG:4326',
        '+proj=longlat +datum=WGS84 +no_defs +type=crs',
        always_xy=True,
        skip_equivalent=True
    )
Expecting nothing
ok
Trying:
    for pt in transproj_eq.itransform([(-2.137, 0.661)]):
        '{:.3f} {:.3f}'.format(*pt)
Expecting:
    '-2.137 0.661'
ok
Trying:
    from pyproj import Transformer
Expecting nothing
ok
Trying:
    transformer = Transformer.from_crs("epsg:4326", "epsg:3857")
Expecting nothing
ok
Trying:
    x3, y3 = transformer.transform(33, 98)
Expecting nothing
ok
Trying:
    "%.3f  %.3f" % (x3, y3)
Expecting:
    '10909310.098  3895303.963'
ok
Trying:
    pipeline_str = (
        "+proj=pipeline +step +proj=longlat +ellps=WGS84 "
        "+step +proj=unitconvert +xy_in=rad +xy_out=deg"
    )
Expecting nothing
ok
Trying:
    pipe_trans = Transformer.from_pipeline(pipeline_str)
Expecting nothing
ok
Trying:
    xt, yt = pipe_trans.transform(2.1, 0.001)
Expecting nothing
ok
Trying:
    "%.3f  %.3f" % (xt, yt)
Expecting:
    '120.321  0.057'
ok
Trying:
    transproj = Transformer.from_crs(
        {"proj":'geocent', "ellps":'WGS84', "datum":'WGS84'},
        "EPSG:4326",
        always_xy=True,
    )
Expecting nothing
ok
Trying:
    xpj, ypj, zpj = transproj.transform(
        -2704026.010,
        -4253051.810,
        3895878.820,
        radians=True,
    )
Expecting nothing
ok
Trying:
    "%.3f %.3f %.3f" % (xpj, ypj, zpj)
Expecting:
    '-2.137 0.661 -20.531'
ok
Trying:
    transprojr = Transformer.from_crs(
        "EPSG:4326",
        {"proj":'geocent', "ellps":'WGS84', "datum":'WGS84'},
        always_xy=True,
    )
Expecting nothing
ok
Trying:
    xpjr, ypjr, zpjr = transprojr.transform(xpj, ypj, zpj, radians=True)
Expecting nothing
ok
Trying:
    "%.3f %.3f %.3f" % (xpjr, ypjr, zpjr)
Expecting:
    '-2704026.010 -4253051.810 3895878.820'
ok
Trying:
    transformer = Transformer.from_proj("epsg:4326", 4326, skip_equivalent=True)
Expecting nothing
ok
Trying:
    xeq, yeq = transformer.transform(33, 98)
Expecting nothing
ok
Trying:
    "%.0f  %.0f" % (xeq, yeq)
Expecting:
    '33  98'
ok
Trying:
    from pyproj import Proj, itransform
Expecting nothing
ok
Trying:
    p1 = Proj('epsg:4326', preserve_units=False)
Expecting nothing
ok
Trying:
    p2 = Proj('epsg:2100', preserve_units=False)
Expecting nothing
ok
Trying:
    points = [(22.95, 40.63), (22.81, 40.53), (23.51, 40.86)]
Expecting nothing
ok
Trying:
    for pt in itransform(p1,p2,points, always_xy=True): '%6.3f %7.3f' % pt
Expecting:
    '411050.470 4497928.574'
    '399060.236 4486978.710'
    '458553.243 4523045.485'
ok
Trying:
    for pt in itransform(4326, 4326, [(30, 60)], skip_equivalent=True):
        '{:.0f} {:.0f}'.format(*pt)
Expecting:
    '30 60'
ok
Trying:
    from pyproj import Proj, transform
Expecting nothing
ok
Trying:
    p1 = Proj('epsg:26915', preserve_units=False)
Expecting nothing
ok
Trying:
    p2 = Proj('epsg:26715', preserve_units=False)
Expecting nothing
ok
Trying:
    x1, y1 = p1(-92.199881,38.56694)
Expecting nothing
ok
Trying:
    x2, y2 = transform(p1,p2,x1,y1)
Expecting nothing
ok
Trying:
    '%9.3f %11.3f' % (x1,y1)
Expecting:
    '569704.566 4269024.671'
ok
Trying:
    '%9.3f %11.3f' % (x2,y2)
Expecting:
    '569722.342 4268814.028'
ok
Trying:
    '%8.3f %5.3f' % p2(x2,y2,inverse=True)
Expecting:
    ' -92.200 38.567'
ok
Trying:
    lats = (38.83,39.32,38.75) # Columbia, KC and StL Missouri
Expecting nothing
ok
Trying:
    lons = (-92.22,-94.72,-90.37)
Expecting nothing
ok
Trying:
    x1, y1 = p1(lons,lats)
Expecting nothing
ok
Trying:
    x2, y2 = transform(p1,p2,x1,y1)
Expecting nothing
ok
Trying:
    xy = x1+y1
Expecting nothing
ok
Trying:
    '%9.3f %9.3f %9.3f %11.3f %11.3f %11.3f' % xy
Expecting:
    '567703.344 351730.944 728553.093 4298200.739 4353698.725 4292319.005'
ok
Trying:
    xy = x2+y2
Expecting nothing
ok
Trying:
    '%9.3f %9.3f %9.3f %11.3f %11.3f %11.3f' % xy
Expecting:
    '567721.149 351747.558 728569.133 4297989.112 4353489.645 4292106.305'
ok
Trying:
    lons, lats = p2(x2,y2,inverse=True)
Expecting nothing
ok
Trying:
    xy = lons+lats
Expecting nothing
ok
Trying:
    '%8.3f %8.3f %8.3f %5.3f %5.3f %5.3f' % xy
Expecting:
    ' -92.220  -94.720  -90.370 38.830 39.320 38.750'
ok
Trying:
    p1 = Proj(proj='latlong',datum='WGS84')
Expecting nothing
ok
Trying:
    x1 = -111.5; y1 = 45.25919444444
Expecting nothing
ok
Trying:
    p2 = Proj(proj="utm",zone=10,datum='NAD27', preserve_units=False)
Expecting nothing
ok
Trying:
    x2, y2 = transform(p1, p2, x1, y1)
Expecting nothing
ok
Trying:
    "%s  %s" % (str(x2)[:9],str(y2)[:9])
Expecting:
    '1402291.0  5076289.5'
ok
Trying:
    from pyproj import CRS
Expecting nothing
ok
Trying:
    c1 = CRS(proj='latlong',datum='WGS84')
Expecting nothing
ok
Trying:
    x1 = -111.5; y1 = 45.25919444444
Expecting nothing
ok
Trying:
    c2 = CRS(proj="utm",zone=10,datum='NAD27')
Expecting nothing
ok
Trying:
    x2, y2 = transform(c1, c2, x1, y1)
Expecting nothing
ok
Trying:
    "%s  %s" % (str(x2)[:9],str(y2)[:9])
Expecting:
    '1402291.0  5076289.5'
ok
Trying:
    xeq, yeq = transform(4326, 4326, 30, 60, skip_equivalent=True)
Expecting nothing
ok
Trying:
    "%.0f %.0f" % (xeq, yeq)
Expecting:
    '30 60'
ok
20 items had no tests:
    pyproj.transformer
    pyproj.transformer.Transformer
    pyproj.transformer.Transformer.__init__
    pyproj.transformer.Transformer.__repr__
    pyproj.transformer.Transformer.__str__
    pyproj.transformer.Transformer.accuracy
    pyproj.transformer.Transformer.area_of_use
    pyproj.transformer.Transformer.definition
    pyproj.transformer.Transformer.description
    pyproj.transformer.Transformer.from_crs
    pyproj.transformer.Transformer.from_pipeline
    pyproj.transformer.Transformer.from_proj
    pyproj.transformer.Transformer.has_inverse
    pyproj.transformer.Transformer.name
    pyproj.transformer.Transformer.to_wkt
    pyproj.transformer.TransformerGroup
    pyproj.transformer.TransformerGroup.__init__
    pyproj.transformer.TransformerGroup.best_available
    pyproj.transformer.TransformerGroup.transformers
    pyproj.transformer.TransformerGroup.unavailable_operations
4 items passed all tests:
  13 tests in pyproj.transformer.Transformer.itransform
  17 tests in pyproj.transformer.Transformer.transform
   6 tests in pyproj.transformer.itransform
  32 tests in pyproj.transformer.transform
68 tests in 24 items.
68 passed and 0 failed.
Test passed.
_________________ test_geometry_area_perimeter__polygon__holes _________________

    @skip_shapely_windows
    def test_geometry_area_perimeter__polygon__holes():
        geod = Geod(ellps="WGS84")
        assert_almost_equal(
            geod.geometry_area_perimeter(
                Polygon(
                    LineString([Point(1, 1), Point(1, 10), Point(10, 10), Point(10, 1)]),
                    holes=[LineString([Point(1, 2), Point(3, 4), Point(5, 2)])],
                )
            ),
>           (-944373881400.3394, 3979008.0359657984),
        )
E       AssertionError: 
E       Arrays are not almost equal to 7 decimals
E       
E       Mismatch: 50%
E       Max absolute difference: 0.00024414
E       Max relative difference: 2.58521153e-16
E        x: array([-9.4437388e+11,  3.9790080e+06])
E        y: array([-9.4437388e+11,  3.9790080e+06])

test/test_geod.py:391: AssertionError
==================== 2 failed, 240 passed in 13.78 seconds =====================
E: pybuild pybuild:341: test: plugin distutils failed with: exit code=1: cd /<<PKGBUILDDIR>>/.pybuild/cpython3_3.7_pyproj/build; python3.7 -m pytest -v --ignore=test/test_datadir.py
dh_auto_test: pybuild --test -i python{version} -p 3.7 returned exit code 13
```

Full buildlog:

 https://buildd.debian.org/status/fetch.php?pkg=python-pyproj&arch=arm64&ver=2.3.0%2Bds-1%7Eexp1&stamp=1566234328&file=log

--- a/pyproj/geod.py
+++ b/pyproj/geod.py
@@ -396,8 +396,8 @@ class Geod(_Geod):
         >>> lons = [-74, -102, -102, -131, -163, 163, 172, 140, 113,
         ...         88, 59, 25, -4, -14, -33, -46, -61]
         >>> poly_area, poly_perimeter = geod.polygon_area_perimeter(lons, lats)
-        >>> "{:.3f} {:.3f}".format(poly_area, poly_perimeter)
-        '13376856682207.406 14710425.407'
+        >>> "{:.1f} {:.1f}".format(poly_area, poly_perimeter)
+        '13376856682207.4 14710425.4'
 
 
         Parameters

--- a/test/test_geod.py
+++ b/test/test_geod.py
@@ -183,7 +183,7 @@ def test_geometry_length__linestring():
     assert_almost_equal(
         geod.geometry_length(LineString([Point(1, 2), Point(3, 4)])),
         313588.39721259556,
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -201,7 +201,7 @@ def test_geometry_length__linestring__radians():
             radians=True,
         ),
         313588.39721259556,
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -213,7 +213,7 @@ def test_geometry_length__linearring():
             LinearRing(LineString([Point(1, 2), Point(3, 4), Point(5, 2)]))
         ),
         1072185.2103813463,
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -225,7 +225,7 @@ def test_geometry_length__polygon():
             Polygon(LineString([Point(1, 2), Point(3, 4), Point(5, 2)]))
         ),
         1072185.2103813463,
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -246,7 +246,7 @@ def test_geometry_length__polygon__radians():
             radians=True,
         ),
         1072185.2103813463,
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -257,7 +257,7 @@ def test_geometry_length__multipolygon():
     assert_almost_equal(
         geod.geometry_length(MultiPolygon([polygon, polygon])),
         2 * 1072185.2103813463,
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -276,7 +276,7 @@ def test_geometry_length__multipolygon__radians():
     assert_almost_equal(
         geod.geometry_length(MultiPolygon([polygon, polygon]), radians=True),
         2 * 1072185.2103813463,
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -287,7 +287,7 @@ def test_geometry_length__multilinestring():
     assert_almost_equal(
         geod.geometry_length(MultiLineString([line_string, line_string])),
         1254353.5888503822,
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -311,7 +311,7 @@ def test_geometry_area_perimeter__linestring():
     assert_almost_equal(
         geod.geometry_area_perimeter(LineString([Point(1, 2), Point(3, 4)])),
         (0.0, 627176.7944251911),
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -329,7 +329,7 @@ def test_geometry_area_perimeter__linestring__radians():
             radians=True,
         ),
         (0.0, 627176.7944251911),
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -341,7 +341,7 @@ def test_geometry_area_perimeter__linearring():
             LinearRing(LineString([Point(1, 2), Point(3, 4), Point(5, 2)]))
         ),
         (-49187690467.58623, 1072185.2103813463),
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -353,7 +353,7 @@ def test_geometry_area_perimeter__polygon():
             Polygon(LineString([Point(1, 2), Point(3, 4), Point(5, 2)]))
         ),
         (-49187690467.58623, 1072185.2103813463),
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -374,7 +374,7 @@ def test_geometry_area_perimeter__polygon__radians():
             radians=True,
         ),
         (-49187690467.58623, 1072185.2103813463),
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -389,6 +389,7 @@ def test_geometry_area_perimeter__polygon__holes():
             )
         ),
         (-944373881400.3394, 3979008.0359657984),
+        decimal=2,
     )
 
 
@@ -399,7 +400,7 @@ def test_geometry_area_perimeter__multipolygon():
     assert_almost_equal(
         geod.geometry_area_perimeter(MultiPolygon([polygon, polygon])),
         (-98375380935.17245, 2144370.4207626926),
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -418,7 +419,7 @@ def test_geometry_area_perimeter__multipolygon__radians():
     assert_almost_equal(
         geod.geometry_area_perimeter(MultiPolygon([polygon, polygon]), radians=True),
         (-98375380935.17245, 2144370.4207626926),
-        decimal=3,
+        decimal=2,
     )
 
 
@@ -429,7 +430,7 @@ def test_geometry_area_perimeter__multilinestring():
     assert_almost_equal(
         geod.geometry_area_perimeter(MultiLineString([line_string, line_string])),
         (-98375380935.17245, 2144370.4207626926),
-        decimal=3,
+        decimal=2,
     )
 
 

